docs/reference/mapping/types/dense-vector.asciidoc

```diff
@@ -212,14 +246,36 @@ properties must be defined.
 The type of kNN algorithm to use. Currently only `hnsw` is supported.
 
 `m`:::
-(Required, integer)
+(Optional, integer)
 The number of neighbors each node will be connected to in the HNSW graph.
 Defaults to `16`.
 
 `ef_construction`:::
-(Required, integer)
+(Optional, integer)
 The number of candidates to track while assembling the list of nearest
 neighbors for each new node. Defaults to `100`.
+
+`quantization_options`:::
+(Optional, object)
+An optional section that configures the quantization configuration. The
+quantization configuration is used to reduce the memory footprint of the
+index. Only `byte` quantization is currently supported and can only be configured if
```

Comment:
Do you think we should add a sentence about disk usage increase with `quantization_options`?